Dockerfile

```diff
@@ -1,34 +1,22 @@
 # Build the manager binary
-FROM golang:1.16 as builder
+FROM golang:1.17 as builder
 
 WORKDIR /workspace
-# Copy the Go Modules manifests
 COPY go.mod go.mod
 COPY go.sum go.sum
-# cache deps before building and copying source so that we don't need to re-download as much
-# and so that source changes don't invalidate our downloaded layer
 RUN go mod download
 
-# Copy the go source
 COPY main.go main.go
 COPY api/ api/
 COPY controllers/ controllers/
-COPY utils/ utils/
-COPY templates/ templates/
-COPY readinessProbe/ readinessProbe/
 
-# Build
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager main.go
 
-# Build readiness probe binary
```

Comment:
For what?

Reply:
@rexagod  We created the readiness prob in the first place so we could communicate the correct status to OCM to facilitate service status on the OCM UI (and let OCM know that the addon is full funcitoning)

I do belive the same reasoning also apply for the Data Federation addon

Reply:
without readiness the add on in a ready state in the installed operator before noobaa is up and running, you will get green tick even if noobaa is not up also, this will create confusion, this is why we need readiness like the name suggest

Reply:
@nb-ohad Can you PLMK what part of OCM UI uses the readiness probe check? Is it the green tick? Or something else entirely? I could have missed it if it was the latter case since I only checked this on OCP.

@naveenpaul1 If there's a green tick even without readiness check, then what difference does the readiness probe, serving readyz 200 status make (I don't see any failed probe checks in the logs anyway)?

Reply:
@rexagod think it in this way, without the readiness prob you will get success mark even if the operator fails to start the noobaa, this is something we don't need, please let me know if you don't get this

Reply:
I didnt get it

Reply:
Resolved over call: Use readiness server (as it was before).